test_room_class.py

```diff
@@ -1,87 +1,152 @@
 import unittest
-from room import Room
+from room import Room, Office
 from people import Person, Fellow, Staff
 from allocator import Dojo
 
-
 class TestRoomClass(unittest.TestCase):
     def setUp(self):
         self.test_dojo = Dojo()
-        self.fellow1 = Fellow()
+        self.fellow1 = Fellow("bryan")
         self.is_office = True
+        self.need_room = True
         self.is_fellow = True
 
+        self.room1 = Room()
+        self.office1 = Office("blue_office")
+        #
         """ creating multiple offices"""
```

Comment:
This should be a comment not a doc string. Doc strings are used to describe what a class/function does.